docs/docs/getting-started/observability.md

```diff
@@ -282,8 +281,14 @@ So let's install new Observability components to help us:
   Scrape metrics from the above DORA metrics endpoint & forward to Prometheus
 - [Grafana](https://grafana.com) (and some prebuilt dashboards): Visualise the data
 
-![add observability](./assets/install01.png)
+**Note**
```

Comment:
Instead of this note, please reorder the items above so that the OTel collector is right below Jaeger and then add in parentheses:
```
(in this guide, we will use the OTel collector that comes bundled with the Jaeger `allInOne` image)
```